include/options_app.h

```diff
@@ -1,13 +1,14 @@
 #ifndef POKEHEARTGOLD_OPTIONS_APP_H
 #define POKEHEARTGOLD_OPTIONS_APP_H
 
+#include "menu_input_state.h"
 #include "options.h"
 #include "overlay_manager.h"
 
 typedef struct OptionsApp_Args {
     u32 unk0;
     Options *options;
-    BOOL *unk8;
+    MenuInputStateMgr *unk8;
```

Comment:
unk8 -> menuInputStateMgr
Also if Mgr = Manager we should write it as Man (shorthand) or straight up in long hand